src/main/perl/FileEditor.pm

```diff
@@ -58,13 +66,45 @@ sub _is_valid_source
     return -f $fn;
 }
 
+sub _is_reference_newer
```

Comment:
After thinking more about it, I prefer to keep it this way: an internal method valid only in the context of the constructor. I added comments (rather than POD) to make this explicit and explain the main attributes used.